src/components/suggestions/SuggestionBox.tsx

```
@@ -0,0 +1,137 @@
+
+                {error && (
+                    <div className="mb-4 bg-red-50 border border-red-200 text-red-700 px-4 py-3 rounded-lg flex items-start space-x-3">
+                        <AlertTriangle className="w-5 h-5 mt-0.5" />
+                        <div>
+                            <p className="font-medium text-sm">Action failed</p>
+                            <p className="text-xs mt-1 text-red-600">{error}</p>
+                        </div>
+                    </div>
+                )}
+
+                <form onSubmit={handleSubmit} className="space-y-4">
```

ESLint

[error] Replace `················` with `········`

(prettier/prettier)